host_core/lib/host_core/providers/builtin/numbergen.ex

```diff
@@ -0,0 +1,17 @@
+defmodule HostCore.Providers.Builtin.Numbergen do
+  def invoke("GenerateGuid", _payload) do
+    Msgpax.pack!(UUID.uuid4())
+  end
+
+  def invoke("RandomInRange", payload) do
```

Comment:
I like this specification of ambiguous things (inclusive range vs exclusive) in the operation name for clarity